doc/lua.yaml

```diff
@@ -185,3 +185,17 @@ values:
   - name: RsvgDimensionData:get()
     desc: |-
       Gets the values of an existing RsvgDimensionData.
+  - name: ret_scale_x,ret_scale_y:cairo_draw_image(file, cs, x, y, scale_x, scale_y)
+    desc: |-
+      Renders an image onto a cairo_surface_t, using imlib2. Returns the amount the image was scaled by
+
+      | Argument            | Description                                                                          |
+      |---------------------|--------------------------------------------------------------------------------------|
+      | file                | Path to the image to render.                                                         |
+      | cs                  | The `cairo_surface_t` to render to.                                                  |
+      | x,y                 | Position to render the image.                                                        |
+      | scale_x, scale_y    | The amount to scale the image, 1.0 provides no scaling, 2.0 for twice the size and   |
+      |                     | 0.5 for half size etc.                                                               |
+      |                     | Default value: No Scaling (1.0,1.0)
+
+      require('cairo_imlib2_helper') in your lua file.                                                  |  
```

Comment:
This formatting is messed up, check the output here: https://deploy-preview-1729--conkyweb.netlify.app/lua#ret_scale_x,ret_scale_y:cairo_draw_image(file,%20cs,%20x,%20y,%20scale_x,%20scale_y)

Reply:
I couldn't use the functions in `logging.h` directly as unfortunately the lua bindings only currently have access to a C compiler.  So I added a simple C re implementation and used that, if this PR is accepted i'll also swap to that for the lua-text PR.

It looks like the docs issue was just a typo.

Reply:
Ah, I guess that makes sense.

Want to try this patch?: https://github.com/brndnmtthws/conky/pull/1740

I think cmake should do the right thing and assume C++ if we simply change the filename.

Reply:
Yep Ill give it a shot when I find some time in the next couple of days.

Reply:
Yep, that's worked thanks